spiketools/sim/trials.py

```diff
@@ -0,0 +1,82 @@
+"""Simulate trials - 2D arrays of event related spiking activity."""
+
+import numpy as np
+
+from spiketools.sim.times import sim_spiketimes_poisson
+from spiketools.utils.checks import check_param_options
+
+###################################################################################################
+###################################################################################################
+
+def sim_trials(n_trials, method=None, time_pre=1, time_post=2, refractory=0.001, **kwargs):
+    """Simulate a collection of trials.
+
+    Parameters
+    ----------
+    n_trials : int
+        Number of trials to simulate.
+    method : {'poisson'}
```

Comment:
Add defaults to params docstring?

Reply:
Do you mean to the `time_pre` / `time_post` arguments in the docstring? That's a good point - though it's not totally obvious the best format for listing both parameters with different defaults - I'll try it out